src/main/java/seedu/address/MainApp.java

```diff
@@ -211,20 +211,26 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
 
     try {
       assignmentAddressBookOptional = storage.readAssignmentAddressBook();
+
+      if (assignmentAddressBookOptional.isPresent()) {
+        logger.info("Data file found!");
+      }
+
       if (!assignmentAddressBookOptional.isPresent()) {
```

Comment:
just small fix here use if else with the above statement instead

Reply:
Okay have edited it accordingly, was using it for debugging